src/components/elements/Notification/constants/baseClasses.ts

```diff
@@ -0,0 +1,8 @@
+const BASE_CLASSES =
+  "flex items-start rounded-lg font-bold tracking-tighter leading-16 p-4 max-w-[35vw] bg-white shadow-[0_0_5px_0_rgba(0,0,0,0.2)] text-bold-body-300";
+export const TYPE_CLASSES = {
+  success: `${BASE_CLASSES} group:text-success-600`,
+  error: `${BASE_CLASSES} group:text-error-600`,
+  warning: `${BASE_CLASSES} group:text-tertiary-600`,
+  default: `${BASE_CLASSES} group:text-success-600`
+};
```

Comment:
This is totally fine, but for future reference, I just meant they should be constants within that same file. No preference that it be moved, this looks good as-is.

Reply:
Oh, so that's what you meant by file-level, sorry I didn't caught it.